docs/release-notes/2.35.0.md

```diff
@@ -19,41 +19,54 @@ Percona Monitoring and Management (PMM) is an open source database monitoring, m
 
 ## Release Highlights
 
-###  Access Control: Limit access to metrics 
+###  Access Control: Limit access to metrics
+
 !!! caution alert alert-warning "Important/Caution"
     Disclaimer:  PMM Access Control is still technical preview and is subject to change. We recommend that early adopters use this feature for testing purposes only.
 
 Starting with PMM 2.35.0, we are excited to announce the first step towards PMM Access Management, a highly anticipated solution by our users. With this solution, PMM allows you to manage who has access to individual Prometheus (Victoria Metics)  metrics based on labels. This new feature provides a standardized method for granting, changing, and revoking access to metrics. With the introduction of label-based access control, you can now establish roles that control which data can be queried based on specific label criteria. It allows you to associate multiple sets of individual Prometheus (Victoria Metics) labels with a role, ensuring that queries will only return data from series that meet the defined labels. This provides you with a powerful and flexible way to manage your data and ensure secure access to the information you need. For more information, visit the Access Control documentation.
 
 PMM Access Management will be introduced in phases to ensure a seamless rollout in line with your requirements and feedback. Stay tuned for further updates on our roadmap for PMM Access Control.
 
-
-###  Deploying PMM in k8 with Helm Chart is now generally available
+###  GA: Deployment of PMM in k8 with Helm Chart
```

Comment:
@dygksy @puneet0191 this is not tested yet

Reply:
@denisok @puneet0191 should I remove it then?

Reply:
@puneet0191 could we complete the test if possible? Customers are waiting its GA for a long time :/